pkg/logicalbackup/logicalbackup.go

```diff
@@ -66,13 +66,61 @@ type LogicalBackuper interface {
 	Wait()
 }
 
+type BackupTables struct {
+	sync.RWMutex
+	data map[dbutils.Oid]tablebackup.TableBackuper
+}
+
+func NewBackupTables() *BackupTables {
+	return &BackupTables{sync.RWMutex{}, make(map[dbutils.Oid]tablebackup.TableBackuper)}
+}
+
+func (bt *BackupTables) GetByOid(oid dbutils.Oid) tablebackup.TableBackuper {
```

Comment:
not sure if you you need that method. you can always skip the second value from the `GetByOidIfExists` if you want `GetByOid` behaviour

Reply:
For symmetry with how the map works (you can assign map index expression to only a single value).